src/Actions/Ledger.ts

```diff
@@ -51,18 +52,23 @@ export class Ledger implements ILedger {
     receipt: any,
     opts: ITransactionOptions,
     from: string,
-    success: boolean
+    success: boolean,
+    paymentModifier: BigNumber
   ): boolean {
     let bounty = new BigNumber(0);
     let cost = new BigNumber(0);
 
+    const gasUsed = new BigNumber(receipt.gasUsed);
```

Comment:
This has to be different in case of success. The reason is that there is a difference in receipt.gasUsed and measuredGasConsumption from the Executed event.

Take a look at the tx https://etherscan.io/tx/0xa30b906f9e2023b071346d0a1e255453c93ce809c3feb8ae775aacf0c1329117

GasUsed by Tx: 104378
MeasuredGasConsumption: 171990 (you can see that in Event Log data[2])

So reimbursement included in bounty (in Executed event data[0]) includes MeasuredGasConsumption * txRequest.gasPrice

I think I mislead you before with calculating the difference in gasPrice.

```
 bounty = new BigNumber(data.slice(0, 66))
        .sub(gasUsed.mul(actualGasPrice))
```